### DISCUSSION TITLE
How to Change Sensors After Initial Configuration

### DISCUSSION THREAD

**Discussion by AndreaFusarPoli** [Alexwijn/SAT#70] [Q&A]
How do I change the sensors (i.e. temperature, umiditiy) after first configuration?

**Comment by Alexwijn** [comment]
You have to reconfigure the integration since those are essential parts of the system.

![image](https://github.com/user-attachments/assets/8bc2c5da-d42a-48a5-bdaa-0fe2284b724b)
